apps/server/src/modules/learnroom/index.ts

```diff
@@ -1,3 +1,4 @@
+export * from './learnroom.config';
```

Comment:
Please make export explicit for module barrel files.

Reply:
Done